exonum/src/messages/protocol.rs

```diff
@@ -552,6 +560,33 @@ impl TransactionsRequest {
     }
 }
 
+/// Request for pool transactions.
```

Comment:
I think that we should add maximum number of transactions to avoid flood attacks.

Reply:
I suppose it should be done in the epic of pool improvements. But I've created the ticket for this task [ECR-3417](https://jira.bf.local/browse/ECR-3417)